tests/unit/test_agent_skill.py

```diff
@@ -1624,6 +1622,12 @@ def test_lint_file_fail_typescript(tmp_path, capsys):
         for i, (result_line, expected_line) in enumerate(
             zip(result_lines, expected_lines)
         ):
-            assert result_line.lstrip('./') == expected_line.lstrip(
-                './'
-            ), f"Line {i+1} doesn't match"
+            if i == 6:
```

Comment:
Should we merge the ts linter first? I believe this belongs to TS linter? Or is this a fix for those linter?

Reply:
The ts linter PR is already merged. But the problem is that not all systems have tsc and/or eslint available, so this is more a "catch all" approach. My WSL installation is different than CLI, for example.